portfolio/src/main/webapp/about.css

```diff
@@ -41,10 +43,25 @@ img {
   justify-content: center;
 } 
 
-.flex a img {
-  margin-right: 20px;
-} 
-
 .last {
   margin-bottom: 30px;
-}
\ No newline at end of file
+}
+
+@media only screen and (max-width: 500px) {
```

Comment:
One of these should probably be set to 499px so it's clear which styles apply at exactly 500. I think as it is right now the latter rules will override the earlier ones.